pkg/pillar/hardware/smart.go

```diff
@@ -115,12 +166,17 @@ func GetInfoFromSATAdisk(diskName string) (*types.DiskSmartInfo, error) {
 	}
 
 	for _, smart := range smartAttrList.Attrs {
+		if smartAttrMap(smart.Id) == "Unknown_Attribute" {
```

Comment:
But wait, you don't report the unknown ones. Seems like a loss of information particularly since SATA SSDs might have a number of additional useful attributed.

Reply:
Thank you, I put this line there to debug. I will remove it.